src/main/java/org/terasology/physicalstats/system/PhysicalStatsSystem.java

```diff
@@ -76,10 +81,11 @@ public void initialise() {
      */
     public boolean updateHealth(EntityRef e, HealthComponent h, PhysicalStatsComponent p)
```

Comment:
You might be able to move the healthComponent inside the method, instead of taking it from the entity before passing it as a param since you have the entity as a param as well